src/wallet/wallet.ts

```diff
@@ -21,20 +21,22 @@ import Input from '../models/input';
 import Address from '../models/address';
 import Network from '../models/network';
 import MineTransaction from './mineTransaction';
+import SendTransactionWalletService from './sendTransaction';
 import { shuffle } from 'lodash';
 import bitcore from 'bitcore-lib';
 import {
   AddressInfoObject,
   GetBalanceObject,
   GetAddressesObject,
   GetHistoryObject,
-  TxProposalUpdateResponseData,
   SendManyTxOptionsParam,
   SendTxOptionsParam,
   WalletStatus,
   Utxo,
   OutputRequestObj,
-  InputRequestObj
+  InputRequestObj,
+  SendTransactionEvents,
+  SendTransactionResponse
```

Comment:
When I started doing this task I created the interface but javascript code does not have any interface implementation. Because of that, we would not be able to use it to enforce the same method returns/signatures on it, then I decided not to do it.

I can create it and the classes on typescript may implement it but the ones still using javascript wouldn't benefit from it.

(This comment is valid for both SendTransaction and HathorWallet classes)

Reply:
I see. I feel that we should do it to explicitly define the interface and understand the impacts of future changes. Regarding the javascript file, I hope it will be migrated to type script and we might benefit from the linter in the future.

Reply:
Makes sense, done here https://github.com/HathorNetwork/hathor-wallet-lib/pull/221/commits/9dd1b760fdfdce10f2f4022fad206998e1777527

Reply:
That's great. Thanks!